physicalplan/scan/function.go

```diff
@@ -68,6 +68,13 @@ func NewFunctionCall(f *parser.Function, selectRange time.Duration) (FunctionCal
 				Metric: labels,
 			}
 		}, nil
+	case "vector":
+		return func(_ labels.Labels, points []promql.Point, stepTime time.Time) promql.Sample {
```

Comment:
Should the labels be reused instead of allocating new ones?

Reply:
Nice catch! Missed updating this.